### ISSUE TITLE
Add a command line option to change the port

### ISSUE THREAD

**Issue by Tazaf** [MediaComem/courses-md#1]
`courses-md` always serves the course on port `3000`, which is a problem when working on several  projects in parallel.

It would be nice to have a `-p` option with the `courses-md` command line to change the serving port:

```bash
$> courses-md -p 3001
```` 
Or automatically find the next available port or something
